workers/src/main/scala/workers/PlotServer.scala

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package workers
+
+import cats.Eval
+import cats.effect.IO
+import cats.effect.IOApp
+import cats.effect.unsafe.implicits.given
+import cats.syntax.all.given
+import explore.events.PlotMessage
+import lucuma.core.enums.Site
+import lucuma.core.enums.TwilightType
+import lucuma.core.math.Coordinates
+import lucuma.core.math.Declination
+import lucuma.core.math.skycalc.solver.ElevationSolver
+import lucuma.core.math.skycalc.solver.Samples
+import lucuma.core.model.Semester
+import lucuma.core.model.TwilightBoundedNight
+import lucuma.core.syntax.boundedInterval.given
+import lucuma.core.syntax.time.given
+import org.typelevel.cats.time.given
+import org.typelevel.log4cats.Logger
+import spire.math.Bounded
+
+import java.time.Duration
+import java.time.Instant
+import java.time.LocalTime
+import scala.ContextFunction1
+import scala.collection.immutable.TreeMap
+import scala.scalajs.js.annotation.JSExport
+import scala.scalajs.js.annotation.JSExportTopLevel
+
+@JSExportTopLevel("PlotServer", moduleID = "exploreworkers")
+object PlotServer extends WorkerServer[IO, PlotMessage.Request] with IOApp.Simple {
```

Comment:
Are we using `IOApp`? What about reloads?

Reply:
Good catch, that was just a test, let me revert.